numpy/core/tests/test_numerictypes.py

```diff
@@ -444,3 +444,14 @@ def test_complex(self, t):
     @pytest.mark.parametrize('t', [np.bool_, np.object_, np.unicode_, np.bytes_, np.void])
     def test_other(self, t):
         assert_equal(np.maximum_sctype(t), t)
+
+
+class TestDocStrings(object):
+    def test_platform_dependent_aliases(self):
+        if sys.flags.optimize > 1:
+            # Docstrings do not get added if PYTHONOPTIMIZE/Py_OptimizeFlag > 1
+            return
```

Comment:
Use `@pytest.mark.skipif(sys.flags.optimize > 1, "no docstrings present to inspect")` or similar here